freqtrade/tests/optimize/test_backtesting.py

```diff
@@ -9,6 +9,13 @@
 from freqtrade.optimize import preprocess
 from freqtrade.optimize.backtesting import backtest, generate_text_table, get_timeframe
 import freqtrade.optimize.backtesting as backtesting
+from freqtrade.strategy.strategy import Strategy
+
+
+def setup_strategy():
```

Comment:
just annotate with `@pytest.fixture` and you don't need to repeat it in every test.